utils/dates-to-epiweek.R

```diff
@@ -4,10 +4,6 @@
 #   "epiweek_full" = TRUE if there are a complete 7 days of data in the epiweek
 #   
 # Example:
-# source("get_us_data.R")
-# daily <- get_us_deaths()
-# daily_date <- dates_to_epiweek(daily)
-# 
 # to only retain the last date of each epiweek (i.e. the forecasting target_end_date),
 #   df %>%
```

Comment:
Nikos has a. package for hub utils he has ported some stuff into including this function. If interested might be worth checking out.